test/upload_wpt_results_test.py

```diff
@@ -366,7 +369,7 @@ def test_failed_request(self):
                                                  override_platform='false',
                                                  total_chunks=2)
 
-        self.assertNotEqual(returncode, 0, stdout)
+        self.assertNotEqual(returncode, 0, stderr)
```

Comment:
This test fails when the command exits successfully. In that case, standard error is less likely to have useful information.